src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -58,11 +60,22 @@ class OutcomeCardContainer extends Component {
     }
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    const curEditingClients = this.state.activeEditingClients;
+    const prevEditingClients = prevState.activeEditingClients;
+    const shouldAnnounceEditing =
+      (curEditingClients.isEmpty() && !prevEditingClients.isEmpty()) ||
+      (!curEditingClients.isEmpty() && prevEditingClients.isEmpty());
+    if (shouldAnnounceEditing) {
+      this.announceEditing(!curEditingClients.isEmpty());
```

Comment:
`announceEditing` is now reactive to `state.activeEditingClients` so we don't need to remember to manually call it.